README.md

```diff
@@ -301,17 +301,80 @@ spec:
 
 You can also achieve persistence by creating a PV in a mounted host folder.
 
-## Mounted Host Folders
-Some drivers will mount a host folder within the VM so that you can easily share files between the VM and host.  These are not configurable at the moment and different for the driver and OS you are using.  Note: Host folder sharing is not implemented in the KVM driver yet.
+## Mounting Host Folders
+`minikube mount /path/to/dir/to/mount` is the recommended way to mount directories into minikube so that they can be used in your local kubernetes cluster.  The command works on all supported platforms.  Below is an example workflow for using `minikube mount`:
 
-| Driver | OS | HostFolder | VM |
-| --- | --- | --- | --- |
-| Virtualbox | Linux | /home | /hosthome |
-| Virtualbox | OSX | /Users | /Users |
-| Virtualbox | Windows | C://Users | /c/Users |
-| VMWare Fusion | OSX | /Users | /Users |
-| Xhyve | OSX | /Users | /Users |
+```
+# terminal 1
+$ mkdir ~/mount-dir
+$ minikube mount ~/mount-dir
+Mounting /home/user/mount-dir/ into /mount-9p on the minikubeVM
+This daemon process needs to stay alive for the mount to still be accessible...
+ufs starting
+# This process has to stay open, so in another terminal...
+```
+
+```
+# terminal 2
+$ echo "hello from host" > ~/mount-dir/hello-from-host
+$ kubectl run -i --rm --tty ubuntu --overrides='
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "ubuntu"
+  },
+  "spec": {
+        "containers": [
+          {
+            "name": "ubuntu",
+            "image": "ubuntu:14.04",
+            "args": [
+              "bash"
+            ],
+            "stdin": true,
+            "stdinOnce": true,
+            "tty": true,
+            "volumeMounts": [{
+              "mountPath": "/mount-9p",
+              "name": "host-mount"
+            }]
+          }
+        ],
+    "volumes": [
+      {
+        "name": "host-mount",
+        "hostPath": {
+          "path": "/mount-9p"
+        }
+      }
+    ]
+  }
+}
+'  --image=ubuntu:14.04 --restart=Never -- bash
```

Comment:
nit take it or leave it: can we just use alpine here or is that going to cause problems?